booking-service/src/test/java/co/axelrod/bookingservice/service/BookingServiceTest.java

```diff
@@ -63,6 +68,23 @@ void confirmBookingTest() {
             - Guest finished booking
             """)
     void happyPath() {
+        CreateBookingRequest request = new CreateBookingRequest();
+        Booking booking_from_request = mapper.map(request);
+        request.setGuestId("guest123");
+        request.setPropertyId("property456");
+        request.setTimestampFrom(System.currentTimeMillis());
+        request.setTimestampTo(request.getTimestampFrom() + 86400000);
+
+        UUID stateMachineUUID = bookingService.createBooking(booking_from_request);
+
+        bookingService.confirmBooking(stateMachineUUID);
```

Comment:
Okay, but it would be better to have an assertion of an actual status, something like:
`assertTrue(Statues.CONFRIMED, bookingService.confirmBooking(stateMachineUUID))`